__tests__/app.test.js

```diff
@@ -86,7 +86,43 @@ describe('GET /api/articles/:article_id', () => {
       .get('/api/articles/not-a-number')
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe('Invalid article ID');
+        expect(body.msg).toBe('Invalid article ID')
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of article objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body
+        expect(Array.isArray(articles)).toBe(true)
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: